tests/tools/test_dist_part.py

```diff
@@ -26,6 +26,15 @@ def _verify_partition_data_types(part_g):
         if k in part_g.edata:
             assert part_g.edata[k].dtype == dtype
 
+def _verify_partition_formats(part_g, formats):
+    # verify saved graph formats
+    if formats is None:
+        assert "coo" in part_g.formats()["created"]
```

Comment:
Where do we introduce the magic word "created"?

Reply:
`DGLHeteroGraph.formats()`

Reply:
In the future we should avoid this type of magic word, instead, either using structured variable, or using global constant.